libgambatte/libretro/libretro_core_options.h

```diff
@@ -989,25 +989,30 @@ struct retro_core_options_v2 options_us = {
 
 #ifndef HAVE_NO_LANGEXTRA
 struct retro_core_options_v2 *options_intl[RETRO_LANGUAGE_LAST] = {
-   &options_us, /* RETRO_LANGUAGE_ENGLISH */
-   NULL,        /* RETRO_LANGUAGE_JAPANESE */
-   NULL,        /* RETRO_LANGUAGE_FRENCH */
-   NULL,        /* RETRO_LANGUAGE_SPANISH */
-   NULL,        /* RETRO_LANGUAGE_GERMAN */
-   NULL,        /* RETRO_LANGUAGE_ITALIAN */
-   NULL,        /* RETRO_LANGUAGE_DUTCH */
-   NULL,        /* RETRO_LANGUAGE_PORTUGUESE_BRAZIL */
-   NULL,        /* RETRO_LANGUAGE_PORTUGUESE_PORTUGAL */
-   NULL,        /* RETRO_LANGUAGE_RUSSIAN */
-   NULL,        /* RETRO_LANGUAGE_KOREAN */
-   NULL,        /* RETRO_LANGUAGE_CHINESE_TRADITIONAL */
-   NULL,        /* RETRO_LANGUAGE_CHINESE_SIMPLIFIED */
-   NULL,        /* RETRO_LANGUAGE_ESPERANTO */
-   NULL,        /* RETRO_LANGUAGE_POLISH */
-   NULL,        /* RETRO_LANGUAGE_VIETNAMESE */
-   NULL,        /* RETRO_LANGUAGE_ARABIC */
-   NULL,        /* RETRO_LANGUAGE_GREEK */
-   NULL,        /* RETRO_LANGUAGE_TURKISH */
+   &options_us,    /* RETRO_LANGUAGE_ENGLISH */
+   &options_ja,    /* RETRO_LANGUAGE_JAPANESE */
+   &options_fr,    /* RETRO_LANGUAGE_FRENCH */
+   &options_es,    /* RETRO_LANGUAGE_SPANISH */
+   &options_de,    /* RETRO_LANGUAGE_GERMAN */
+   &options_it,    /* RETRO_LANGUAGE_ITALIAN */
+   &options_nl,    /* RETRO_LANGUAGE_DUTCH */
+   &options_pt_br, /* RETRO_LANGUAGE_PORTUGUESE_BRAZIL */
+   &options_pt_pt, /* RETRO_LANGUAGE_PORTUGUESE_PORTUGAL */
+   &options_ru,    /* RETRO_LANGUAGE_RUSSIAN */
+   &options_ko,    /* RETRO_LANGUAGE_KOREAN */
+   &options_cht,   /* RETRO_LANGUAGE_CHINESE_TRADITIONAL */
+   &options_chs,   /* RETRO_LANGUAGE_CHINESE_SIMPLIFIED */
+   &options_eo,    /* RETRO_LANGUAGE_ESPERANTO */
+   &options_pl,    /* RETRO_LANGUAGE_POLISH */
+   &options_vn,    /* RETRO_LANGUAGE_VIETNAMESE */
+   &options_ar,    /* RETRO_LANGUAGE_ARABIC */
+   &options_el,    /* RETRO_LANGUAGE_GREEK */
+   &options_tr,    /* RETRO_LANGUAGE_TURKISH */
+   &options_sk,    /* RETRO_LANGUAGE_SLOVAK */
+   &options_fa,    /* RETRO_LANGUAGE_PERSIAN */
+   &options_he,    /* RETRO_LANGUAGE_HEBREW */
+   &options_ast,   /* RETRO_LANGUAGE_ASTURIAN */
+   &options_fi,    /* RETRO_LANGUAGE_FINNISH */
```

Comment:
This here is what causes the build issue - it will fix itself, once the sync cycle runs.